Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
@xu-cheng If we leave as a comment will that mean we will still be building with Java 1.8? 

Also I agree with xu-cheng about the build/run number this is only a problem because Kafka doesn't set sourceCapability flag inside it's build.  Once Kafka fixes issue https://issues.apache.org/jira/browse/KAFKA-2034 we can go back to :java => "1.7+"